src/fu-util.c

```diff
@@ -3975,13 +3941,14 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Gets the list of updates for connected hardware"),
 			      fu_util_get_updates);
-	fu_util_cmd_array_add(cmd_array,
-			      "update,upgrade",
-			      /* TRANSLATORS: command argument: uppercase, spaces->dashes */
-			      _("[DEVICE-ID|GUID]"),
-			      /* TRANSLATORS: command description */
-			      _("Updates all firmware to latest versions available"),
-			      fu_util_update);
+	fu_util_cmd_array_add(
+	    cmd_array,
+	    "update,upgrade",
+	    /* TRANSLATORS: command argument: uppercase, spaces->dashes */
+	    _("[DEVICE-ID|GUID]"),
+	    /* TRANSLATORS: command description */
+	    _("Update all devices if unspecified, or particular selection(s) if specified"),
```

Comment:
Maybe `Updates all specified devices to latest firmware version, or all devices if unspecified`?